src/lib/jobs/check_subscriptions.rs

```diff
@@ -41,44 +42,55 @@ pub async fn fetch_and_process_new_subscriptions(
     // Get all results from bigquery table that stores new subscription reports
     let query = "SELECT * FROM `cjms_bigquery.subscriptions_v1`;";
     let mut rs = bq.get_bq_results(query).await;
-    rs.report_stats(statsd, &TraceType::CheckSubscriptions);
+    rs.report_stats(statsd, &LogKey::CheckSubscriptions);
     while rs.next_row() {
         // If can't deserialize e.g. required fields are not available log and move on.
         let mut sub = match make_subscription_from_bq_row(&rs) {
             Ok(sub) => {
-                // TODO - LOGGING
-                println!(
-                    "Successfully deserialized subscription from bq row: {}",
-                    sub.id
+                info_and_incr!(
+                    statsd,
+                    LogKey::CheckSubscriptionsDeserializeBigQuery,
+                    subscription_id = sub.id.to_string().as_str(),
+                    "Successfully deserialized subscription from BigQuery row",
                 );
                 sub
             }
             Err(e) => {
-                // TODO - LOGGING - Log information and get a metric
-                println!(
-                    "Failed to make subscription for bq result row. {:?}. Continuing...",
-                    e
+                error_and_incr!(
+                    statsd,
+                    LogKey::CheckSubscriptionsDeserializeBigQueryFailed,
+                    error = e,
+                    "Failed to make subscription for BigQuery result row. Continuing...",
                 );
                 continue;
             }
         };
         let (aic, aic_found_in_archive) = match aics.fetch_one_by_flow_id(&sub.flow_id).await {
             Ok(aic) => {
-                // TODO - LOGGING
-                println!("Succesfully fetched aic: {}.", aic.id);
+                info_and_incr!(
+                    statsd,
+                    LogKey::CheckSubscriptionsAicFetch,
+                    aic_id = aic.id.to_string().as_str(),
+                    "Successfully fetched aic",
+                );
                 (aic, false)
             }
             Err(_) => match aics.fetch_one_by_flow_id_from_archive(&sub.flow_id).await {
                 Ok(aic) => {
-                    // TODO - LOGGING - Note that we had to pull from archive table
-                    println!("AIC {} was retrieved from archive table.", aic.id);
+                    info_and_incr!(
+                        statsd,
+                        LogKey::CheckSubscriptionsAicFetchFromArchive,
+                        aic_id = aic.id.to_string().as_str(),
+                        "Successfully fetched aic from archive table",
```

Comment:
tweaking back to original message to emphasize that it came, somewhat unexpectedly, from the archive table.